config/config.default.js

```diff
@@ -200,7 +200,9 @@ var path = require('path'),
                 }]
             },
             extlibExcludes: ['config/config..*.js$'],
-            behindSecureProxy: false
+            behindSecureProxy: false,
+            // Configuration for bodyParser.json, can be fully customized by the user
+            json: {}
```

Comment:
the name should be bodyParser.json instead of just json.
json is too generic and we might want to expose other types of config for the bodyParser later.
sorry if I was not clear enough in my previous review

Reply:
Not a problem, it is better this way, and more extensible to other middleware as well.